plugins/source/gcp/client/client.go

```diff
@@ -369,3 +385,51 @@ func setUnion(a []string, b []string) []string {
 	}
 	return union
 }
+
+func (c *Client) configureEnabledServices() error {
+	var esLock sync.Mutex
+	g, ctx := errgroup.WithContext(context.Background())
+	maxGoroutines := 10
+	goroutinesSem := semaphore.NewWeighted(int64(maxGoroutines))
+	for _, p := range c.projects {
+		project := p
+		if err := goroutinesSem.Acquire(ctx, 1); err != nil {
+			return err
+		}
+		g.Go(func() error {
+			defer goroutinesSem.Release(1)
+			cl := c.withProject(project)
+			svc, err := cl.fetchEnabledServices(ctx)
+			esLock.Lock()
+			c.EnabledServices[project] = svc
```

Comment:
If `fetchEnabledServices` fails we set `nil` svc for the project here. If the `Configure` is turned into a warning later (instead of stopping execution) this will present an issue in the form of a panic.

Reply:
Why would it panic? we are checking both for not nil and for boolean == true:
`cl.EnabledServices[projectId] != nil && cl.EnabledServices[projectId][enabledService]`

Reply:
Oh ok didn't see that 👍